outputs/client.go

```diff
@@ -123,6 +128,15 @@ func (c *Client) Post(payload interface{}) error {
 		req.Header.Add("Authorization", "GenieKey "+c.Config.Opsgenie.APIKey)
 	}
 
+	if c.OutputType == Kubeless {
+		b := make([]byte, 11)
+		rand.Read(b)
+		base64.RawURLEncoding.EncodeToString(b)
```

Comment:
this is needed?

Reply:
> lgtm, just some nits:
> 
>     * squash the commits to have just one commit
> 
>     * and just a comment, try to avoid to update dependencies in the PR that does other thing, if the PR need a new dependency is fine, but avoid update non-releated dependencies, we can open a separated PR for that

I squashed the commits, you're right.

For the dependencies, I faced big conflicts while I was developping (in fact, I extracted some snippets from `kubeless cli` before rewriting from scratch everything), and that's a relicate. You're right 😉 . Sorry.